bot/constants.py

```diff
@@ -1,10 +1,26 @@
 import pathlib
 import os
+import typing
 
 # env vars
 PREFIX = os.getenv("PREFIX") or "!"
 TOKEN = os.getenv("TOKEN")
 
+
+class Roles(typing.NamedTuple):
+    ANNOUNCEMENTS_ID = int(os.getenv("ANNOUNCEMENTS_ID", "789978290844598272"))
+    POLLS_ID = int(os.getenv("POLLS_ID", "790043110360350740"))
+
+
+class Emojis(typing.NamedTuple):
+    CONFIRMATION_EMOJI = "<:yes:794231332770938901>"
+    WARNING_EMOJI = "\u26a0"
+
+
+class Colors(typing.NamedTuple):
+    GREEN = 0x32A05A
```

Comment:
```suggestion
    green = 0x32A05A
```
Sorry had suggested it to be upper case green last time.